docs/contributing/index.md

```diff
@@ -63,29 +63,54 @@ See {doc}`setup-build` for instructions for how to set up and build the document
-- Create a new issue in [`documentation`](https://github.com/plone/documentation/issues), [`plone.api`](https://github.com/plone/plone.api/issues), [`plone.restapi`](https://github.com/plone/plone.restapi/issues), or [`volto`](https://github.com/plone/volto/issues).
-- Discuss during conferences, trainings, and other Plone events.
-- Ask on the [Plone Community Forum, Documentation topic](https://community.plone.org/c/documentation/13).
-- Ask in the [Plone chat on Discord](https://discord.com/invite/zFY3EBbjaj).
+After your Plone Contributor Agreement has been approved, and you have been added to the Plone GitHub organization as a member of the [Developers](https://github.com/orgs/plone/teams/developers) team, we recommend that you follow our guidelines.
 
-As a convenience, at the top right of every page, there is a GitHub navigation menu.
-Tap, click, or hover over the GitHub Octocat icon for options.
+1.  Discuss whether you should perform any work.
+    Any method below is acceptable, but are listed in order of most likely to get a response.
 
-```{image} /_static/github-navigation.png
-:alt: GitHub navigation menu 
-```
+    -   Search for open issues in the issue trackers of documented projects, and comment on them.
+        If an issue does not already exist for what you want to work on, then create a new issue in its issue tracker.
+
+        As a convenience, at the top right of every page, there is a GitHub navigation menu.
+        Tap, click, or hover over the GitHub Octocat icon for options.
+
+        ```{image} /_static/github-navigation.png
+        :alt: GitHub navigation menu 
+        ```
+
+        You can use this menu to quickly navigate to the `documentation` source repository or open an issue.
+        Note that this navigation convenience is provided only for the `documentation` repository.
+
+        -   [`documentation`](https://github.com/plone/documentation/issues)
+        -   [`plone.api`](https://github.com/plone/plone.api/issues)
+        -   [`plone.restapi`](https://github.com/plone/plone.restapi/issues)
+        -   [`volto`](https://github.com/plone/volto/issues)
+
+    -   Discuss during conferences, trainings, and other Plone events.
+    -   Ask on the [Plone Community Forum, Documentation topic](https://community.plone.org/c/documentation/13).
+    -   Ask in the [Plone chat on Discord](https://discord.com/invite/zFY3EBbjaj).
+
+1.  For first-time contributors especially, **you do not need to ask to be assigned to, or to work on, an open issue**.
+    As in any open source project, you can start work on open issues at your convenience.
+
+    ```{tip}
+    Working on documentation or on issues labeled with either `33 needs: docs` or `41 lvl: easy` are the two best ways for first-timers to contribute.
```

Comment:
```suggestion
    Working on documentation or on issues labeled with either `31 needs: help`, `33 needs: docs` or `41 lvl: easy` are the two best ways for first-timers to contribute.
```

Reply:
Oops, actually, no. I need to revert this commit because this tip is for newbies who need this direction. `31 needs: help` can be too advanced for newbies.